bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -206,7 +211,7 @@ case object SemesterConverter2018ATo2018B extends SemesterConverter {
 
   val (texesRemover, texesRemoverMessage) = removeBlueprint("texes", "Texes")
 
-  override val transformers = List(gmosnAltairRemover, dssiGNToAlopeke, phoenixSite, texesRemover)
+  override val transformers = List(dssiGNToAlopeke, phoenixSite, texesRemover)
```

Comment:
Why is this removed? Because Altair won't be available for any instrument rather that only for gmosN?

Reply:
Yes, exactly that. It seems like a pointless check at this point.

I was hemming and hawing about removing it entirely since it only applied to that one semester and probably won't be applicable moving forward.

Reply:
Fair enough